packages/pancake-profile-sdk/src/constants/teams.ts

```diff
@@ -0,0 +1,54 @@
+import { Team } from "../types";
+
+const teams: Team[] = [
+  {
+    id: 1,
+    name: "Syrup Storm",
+    description: "The storm's a-comin! Watch out! These bulls are stampeding in a syrupy surge!",
+    images: {
+      lg: "syrup-storm-lg.png",
+      md: "syrup-storm-md.png",
+      sm: "syrup-storm-sm.png",
+      alt: "syrup-storm-alt.png",
```

Comment:
It feels like we're creating a bit of a weird cross-dependency by having images within the FE, depended on by the SDK?

Initially when I saw these relative images (`teams`, `nfts`, `campaigns` constants)... I thought maybe they should also be bundled with the SDK? I guess there would be awful bundle-size implications for that though?

If our plan is to actually rip out as much of the profiles code as possible within the FE, and instead use the SDK... then maybe these images _should_ be within the SDK... or at least independent of the FE?

If we're keeping it as-is, then maybe it's worth highlighting in the docs how an SDK-user would construct the image path relative to our domain to access these?

Reply:
Actually great point, I almost forgot about it, thanks!
Yeah, I was thinking bundling images with SDK, they are not that heavy so it should be fine and reduces stress on SDK consumer.